src/libwrap/Makefile.am

```diff
@@ -9,18 +9,22 @@ libgrate_wrap_la_CPPFLAGS += -DENABLE_RNN
 endif
 
 libgrate_wrap_la_SOURCES = \
+	adler32.h \
 	cdma_parser.c \
 	cdma_parser.h \
 	cgdrv.c \
 	disasm.h \
 	disasm.c \
 	drm.h \
+	drm_fourcc.h \
```

Comment:
Doesn't look like this is the commit that adds drm_fourcc.h, is this in the wrong commit perhaps?

Reply:
Yep